<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>623</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="te_desxifrat">
       <property name="placeholderText">
        <string>Text Desxifrat</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Clau</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_clau"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="bt_xifrar">
         <property name="text">
          <string>Xifrar&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_desxifrar">
         <property name="text">
          <string>&lt;Desxifrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_majuscules">
         <property name="text">
          <string>May√∫scules</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="te_xifrat">
       <property name="placeholderText">
        <string>Text Xifrat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
